internal/app/upstream/client.go

```diff
@@ -134,11 +134,11 @@ func New(
 	}, nil
 }
 
-func (m *client) OpenStream(request transport.Request) (<-chan transport.Response, func()) {
+func (m *client) OpenStream(request transport.Request, aggregatedKey string) (<-chan transport.Response, func()) {
```

Comment:
can we just run the request through the mapper rather than modifying the interface?

Reply:
i think that cost is not necessary compared to the interface change

Reply:
i understand it looks awkward to introduce the param for just logging...i also think introducing dependency on mapper is similarly awkward. The client is supposed to be low level and should not contain the dependencies from orchestrator